crud.c

```diff
@@ -0,0 +1,179 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+// structure of user
+typedef struct {
+    int id;
+    char name[50];
+    int age;
+} User;
+
+// creates file to store user data
+void createFile() {
+    FILE *file = fopen("users.txt", "w");
+    if (file == NULL) {
+        printf("Error in creating the file.\n");
+        return;
+    }
+    fclose(file);
+}
+
+// Function to add a user
+void addUser() {
+    User user;
+    FILE *file = fopen("users.txt", "a");
+    if (!file) {
+        printf("Error in opening the file.\n");
+        return;
+    }
+
+    printf("Enter user ID: ");
+    scanf("%d", &user.id);
+    printf("Enter user name: ");
+    getchar();
+    fgets(user.name, sizeof(user.name), stdin);
+    user.name[strcspn(user.name, "\n")] = '\0';
+    printf("Enter user age: ");
+    scanf("%d", &user.age);
+
+    fprintf(file, "%d,%s,%d\n", user.id, user.name, user.age);
+    fclose(file);
+    printf("User added successfully.\n");
+}
+
+// Function to display all users
+void displayUsers() {
+    FILE *file = fopen("users.txt", "r");
+    if (!file) {
+        printf("Error opening the file.\n");
+        return;
```

Comment:
Don't use unnecessary return. Adjust your logic to handle the negative scenario.

Reply:
Okay!